src/adapters/aol.js

```diff
@@ -82,6 +69,9 @@ var AolAdapter = function AolAdapter() {
       return _addErrorBid(response, context);
     }
 
+    // clean up--we no longer need to store the bid
+    delete bidmanager.pbCallbackMap[context.alias];
```

Comment:
not really related to this PR, but I think it'd be better if we changed bidmanager to better encapsulate the `pbCallbackMap`, eg. adding `setPlacementIdByCBIdentifier` and `removePlacementByCBIdentifier`